lib/reform/form/active_model.rb

```diff
@@ -4,8 +4,11 @@ def self.included(base)
       extend ClassMethods
       register_feature ActiveModel
 
-      extend Forwardable
-      def_delegators :model, :persisted?, :to_key, :to_param, :id
+      %i(persisted? to_key to_param id).each do |method|
+        define_method method do
+          model.public_send(method)
```

Comment:
Can we not simply use https://www.rubydoc.info/gems/activesupport/Module:delegate as you suggested in the first place?

Reply:
You're right, I updated the code to use rails' `delegate` method instead.